src/content/guides/asset-management.md

```diff
@@ -146,7 +146,7 @@ Entrypoint main = bundle.js
 ...
 ```
 
-Open up `index.html` in your browser again and you should see that `Hello webpack` is now styled in red. To see what webpack did, inspect the page (don't view the page source, as it won't show you the result) and look at the page's head tags. It should contain our style block that we imported in `index.js`.
+Open up `index.html` in your browser again and you should see that `Hello webpack` is now styled in red. To see what webpack did, inspect the page (don't view the page source, as it won't show you the result, because this `style` tag is dynamically created by javascript) and look at the page's head tags. It should contain our style block that we imported in `index.js`.
```

Comment:
```suggestion
Open up `index.html` in your browser again and you should see that `Hello webpack` is now styled in red. To see what webpack did, inspect the page (don't view the page source, as it won't show you the result, because the `style` tag is dynamically created by JavaScript) and look at the page's head tags. It should contain our style block that we imported in `index.js`.
```